api.ts

```diff
@@ -144,25 +144,18 @@ export interface Block {
   view_change_proofs: string[];
 }
 
-export type Transaction =
-  | Tagged<"Committed", CommittedTransaction>
-  | Tagged<"Rejected", RejectedTransaction>;
-
-export interface CommittedTransaction {
-  hash: string
-  /**
-   * WIP zeroed
-   */
+export interface TransactionDTO {
+  hash: string;
   block_hash: string;
   payload: TransactionPayload;
   signatures: Signature[];
-}
-
-export interface RejectedTransaction extends CommittedTransaction {
-  /**
-   * List of serialized {@link @iroha2/data-model#TransactionRejectionReason}
+ /**
+   * If the transaction results in an error, this field will contain the rejection reason as a string.
+   * If the transaction is successful, this field will not be present (undefined).
+   * A stringified rejection reason indicates that the transaction is rejected.
+   * The absence of this field indicates that the transaction is committed.
```

Comment:
This docstring contains duplication. Consider one of the following changes:

```suggestion
   * A stringified rejection reason indicates that the transaction is rejected.
   * The absence of this field indicates that the transaction is committed.
```

```suggestion
   * If the transaction results in an error, this field will contain the rejection reason as a string.
   * If the transaction is successful, this field will not be present (undefined).
```